pkg/restore/dataupload_retrieve_action.go

```diff
@@ -29,19 +29,22 @@ import (
 
 	velerov1api "github.com/vmware-tanzu/velero/pkg/apis/velero/v1"
 	velerov2alpha1 "github.com/vmware-tanzu/velero/pkg/apis/velero/v2alpha1"
+	velerov1client "github.com/vmware-tanzu/velero/pkg/generated/clientset/versioned/typed/velero/v1"
```

Comment:
Let's use the controller runtime client instead as we are moving away from the generated clients

Reply:
Done the changes, thanks.